### PR TITLE
TE-366 Tidy styles build step

### PR DESCRIPTION
**Pull request by marcdavi-es**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-366)

### What **one** thing does this PR do?
Removes `gulp` from the project and hands over styles build to webpack.

### Any other notes
Advantages to this setup:
- Fewer files. We don't need the `semantic-ui` install in the project. Less files are imported from `semantic-ui-less` in `node_modules`.
- One less build step. `npm run integration:pr` now runs in ~25s down from >1min
- Less tooling. Without `semantic-ui` dependencies we have ~150 fewer packages in the project.
- Consistency - our whole build is handled by the same tooling. 

This PR also includes a handful of minor fixes to styles broken by the change of styles ordering. (My fault for using alphabetical order rather than canonical semantic ui order for the gulp build)

### CODE REVIEW

File: src/components/general-widgets/Review/component.js [hunk 1]
```diff
@@ -40,10 +40,9 @@ export const Component = ({
               </Subheading>
             </GridColumn>
             <GridColumn
-              mobile={5}
+              mobile={12}
```
[the remark this answers is not in the document]
Comment (marcdavi-es) [reply src/components/general-widgets/Review/component.js]: You got me.

[the remark this answers is not in the document]
Comment (Jonnygwi) [reply src/components/general-widgets/Review/component.js]: +1

File: package.json [hunk 2]
```diff
@@ -61,16 +58,17 @@
     "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-react": "^7.7.0",
     "file-loader": "^1.1.9",
-    "gulp": "^3.9.1",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^22.4.3",
+    "less": "^2.7.3",
```
[the remark this answers is not in the document]
Comment (marcdavi-es) [reply package.json]: Si - `less-loader` needs `less`

File: src/components/general-widgets/Review/component.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcdavi-es) [reply src/components/general-widgets/Review/component.js]: Done.

File: styleguide/webpack.config.js [hunk 3]
```diff
@@ -2,33 +2,39 @@ const path = require('path');
...
-        include: INCLUDE_PATHS,
+        include: JS_INCLUDE_PATHS,
         loader: 'babel-loader',
       },
       {
         test: /\.css$/,
-        include: INCLUDE_PATHS,
-        use: ['style-loader', 'css-loader'],
+        include: path.join(ROOT_PATH, 'src/styles'),
+        use: ['style-loader', 'css-loader', 'less-loader'],
```
[the remark this answers is not in the document]
Comment (marcdavi-es) [reply styleguide/webpack.config.js]: For another ticket/PR?

**Pull request merged**